parcel-bundler/parcel
# adding link-all and unlink-all scripts

# ↪️ Pull Request

This provides two new scripts and commands in package.json

- **link-all** runs "yarn install", "yarn unlink" and "yarn link" for every package in the packages directory. and then runs "yarn link [dependency]" for every dependency that is also within the packages directory.
- **unlink-all** runs "yarn unlink" and deletes the node_modules directory for every package in the packages directory.

## 💻 Examples




## 🚨 Test instructions

1. Run "yarn run link-all"
2. Check several packages and verify that their node_modules dependencies have been installed and they are correctly symlinked to each other.
3. Run "yarn run unlink-all"
4. Check several packages and verify that their node_modules has been removed. You can also try to link to one of the packages from another project and verify that it is no longer available with "yarn link packagename"

## ✔️ PR Todo
